README.md

```diff
@@ -20,6 +20,8 @@ The agent can act as a client or as a server if the host is placed in a group ca
 * `consul_bootstrap_expect`: When `consul_retry_join` is not defined, this provides the number of expected servers in the datacenter.
 * `consul_ui_enabled`: Whether to enable the UI or not, default is false.
 * `consul_node_meta`: This object allows associating arbitrary metadata key/value pairs with the local node, which can then be used for filtering results from certain catalog endpoints.
+* `consul_tag_server`: set to the name of the group that will be consul servers. Default 'consul'.
+* `consul_command`: Full path to the consul command. Default: `/usr/local/bin/consul`
```

Comment:
I would prefer a `consul_install_dir` variable which can be re-used in the `unarchive` and set to `/usr/local/bin` by default.